Cargo.toml

```diff
@@ -72,15 +72,17 @@ version = "0.16.0"
 [profile.release]
 # debug = 1 means line charts only, which is minimum needed for good stack traces
 debug = 1
+# Write debug info into a separate file.
 split-debuginfo = 'packed'
```

Comment:
Looking on the documentation I would expect the debug info to be written to the separate file https://doc.rust-lang.org/rustc/codegen-options/index.html#split-debuginfo

so it seems like it doesn't work as expected?

Reply:
The debug info is needed in a separate file. But the debug info in the binary is not stripped, if we explicitly specify `strip = 'none'`